tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb

```diff
@@ -59,7 +71,15 @@ def self.available_options
             UI.user_error!("Symbol maps file needs to be a directory containing symbol map files")
           end
         end
+
+        options = {}
+        options = options_from_info_plist(default_info_plist_path) if file_path = default_info_plist_path
         [
+          FastlaneCore::ConfigItem.new(key: :api_key,
+                                       env_name: "BUGSNAG_API_KEY",
+                                       description: "Bugsnag API Key",
+                                       optional: true,
+                                       default_value: options[:apiKey]),
```

Comment:
Would it be a good idea to add some validation here?  How does the upload server react if an invalid API key is used?

Reply:
I've added some default validation, but this doesn't always trigger depending on where the api key is provided. The upload server responds with invalid apikey and fails the upload, so hopefully this should be enough to indicate whats gone wrong